integration/tests/basic_functionality/negative/neg_deploy_with_missing_r10k_yaml.rb

```diff
@@ -0,0 +1,45 @@
+require 'git_utils'
+require 'r10k_utils'
+require 'master_manipulator'
+test_name 'CODEMGMT-84 - C59118 Attempt to Deploy with Missing r10k Configuration File'
+
+#Init
+env_path = on(master, puppet('config print environmentpath')).stdout.rstrip
+git_repo_path = '/git_repos'
+git_control_remote = File.join(git_repo_path, 'environments.git')
+
+r10k_config_path = '/etc/r10k.yaml'
+r10k_config_bak_path = "#{r10k_config_path}.bak"
+
+#In-line files
+r10k_conf = <<-CONF
+sources:
+  broken:
+    basedir: "#{env_path}"
+    remote: "#{git_control_remote}"
+CONF
+
+#Verification
+error_message_regex = /Error while running.*R10K\:\:Deployment.*No configuration file/
+
+#Teardown
+teardown do
+  step 'Restore Original "r10k" Config'
+  on(master, "mv #{r10k_config_bak_path} #{r10k_config_path}")
+end
+
+#Setup
+step 'Backup Current "r10k" Config'
+on(master, "mv #{r10k_config_path} #{r10k_config_bak_path}")
+
+step 'Update the "r10k" Config'
+create_remote_file(master, r10k_config_path, r10k_conf)
```

Comment:
This isn't necessary since an r10k config already exists on the system. The pre-suites ensure a working r10k config is laid down before testing begins.